client/blocks/post-share/index.jsx

```diff
@@ -515,7 +515,7 @@ class PostShare extends Component {
 		if ( ! hasRepublicizeFeature ) {
 			return (
 				<div>
-					<UpgradeToPremiumNudge { ...this.props } />
+					<UpgradeToPremiumNudge siteId={ siteId } />
```

Comment:
It's awesome to see how the interface of this component has become minimal. Simplification + turning implicits into explicits like this tends to mean we are doing the right thing :)